README.md

```diff
@@ -10,13 +10,26 @@ database versions, including Oracle.
 
 ## Quickstart
 
-Clone this repository somewhere. Also make sure you have docker and docker-compose installed.
+Clone this repository and ensure you have docker and docker-compose installed.
 
-Ensure that the `DJANGO_PATH` variable points to the root of the Django repo:
+### Environment
+
+You must set the `DJANGO_PATH` environment variable to the path of your local Django installation.
+This can either be added to the `.env` file, or interactively set in your shell, like so:
 
 `export DJANGO_PATH=~/projects/django/`
 
-If you see a docker-compose warning about it not being defined followed by an error ensure that is defined in the shell you are using.
+You must also check the `.env` file (in the root of this repository) and ensure that the versions
+of both Python and and database(s) you wish to test against are properly configured. For example,
+Django 5.0 requires Python 3.10 (or later) and PostgreSQL 13 (or later), so if you wish to
+test against Django 5.0 with PostgreSQL, an appropriate `.env` file may be:
+
+```env
+PYTHON_VERSION=3.10
+POSTGRES_VERSION=13
+```
+
```

Comment:
I'm not sure this is a `must`.

While `DJANGO_PATH` will likely rarely change it's not uncommon to have to test against different versions of Python and databases for the same changes. I usually just pass them directly in the docker-compose invocation instead

```
PYTHON_VERSION=3.11 POSTGRES_VERSION=15 docker-compose run postgres --noinput --parallel 1 --failfast --debug-sql db_functions.comparison.test_json_object.JSONObjectTests
```

All that to say that this is likely not a _must_.

Reply:
@charettes Thanks for reviewing! The reason I inserted this is because, currently, the test suite does not run against `Django 5.1-dev` with the default `.env` settings - it must be updated so that `PYTHON_VERSION=3.10` and `POSTGRES_VERSION=13`.

There wasn't any reference to this in the README already. Admittedly, it does seem very obvious that the tests must be run using a supported Python version, but trying to do otherwise throws an obscure and unhelpful error and so I thought it may be helpful to highlight early on so as to avoid people potentially troubleshooting the issue when expecting it to 'just work'. Are you proposing changing `must` to `may wish to` or `should`, or removing the section all together?

Reply:
Or, I suppose, another solution might just be to focus the entire README on using the environment variables inline, rather than relying on the defaults and the `.env` file as it currently does.

It does seem like using the environment variables inline, as you suggest, is the most sensible solution - perhaps I could rewrite all the suggested commands with that in mind?

Reply:
Ahh that makes sense, I missed that!

It seems that checking in an `.env` file is kind of an anti-pattern in the first place but maybe we could adapt it to use `PYTHON_VERSION=3.10` and `POSTGRES_VERSION=13` so it's always kept in sync with the latest minimum supported versions of Python and the respective database backends in `django/django@main`?